processing-service-core/api/swagger/swagger.yaml

```diff
@@ -1254,8 +1469,8 @@ definitions:
         description: The value for the global config key.  
       scope:  
         type: string  
-        description: Defines the scope of use for the global config key. If the scope is set to 'TENANT' then it may be overridden by tenant configurations.
-        enum: ["TENANT"]
+        description: Defines the scope of use for the global config key. If the scope is set to 'TENANT' then it may be overridden by tenant configurations. Same thing for the 'RESPOSITORY' one, but at the repository level.
```

Comment:
Rather than `Same thing for the 'RESPOSITORY' one, but at the repository level.` write `If the scope is set to 'RESPOSITORY' then it may be overridden either by tenant configurations or by repository configurations.`